src/test/java/org/datadog/jmxfetch/TestCommon.java

```diff
@@ -270,7 +270,7 @@ public void assertMetric(
                     assertEquals(countTags, mTags.size());
                 }
                 for (String t : tags) {
-                    assertTrue(mTags.contains(t));
+                    assertTrue(t + " missing from metric tags", mTags.contains(t));
```

Comment:
```suggestion
                     assertThat(mTags, hasItem(t));
```

You'll have to add the following to you imports:

```
import static org.hamcrest.CoreMatchers.hasItem;
import static org.hamcrest.MatcherAssert.assertThat;
```